models/app.models.js

```diff
@@ -43,4 +43,35 @@ exports.patchReviewVotesById = (review_id, inc_votes) => {
         }
         return review
     })
+}
+
+exports.selectReviews = (category) => {
+    const allowedCategory = ['euro game', 'social deduction', 'dexterity', "children's games"];
```

Comment:
Hard coding the categories will cause problems if we add more categories later - you should get this information from the database